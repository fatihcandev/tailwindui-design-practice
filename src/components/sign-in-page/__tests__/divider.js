import React from 'react'
import renderer from 'react-test-renderer'

import Divider from '../divider'

describe('Divider', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Divider />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
