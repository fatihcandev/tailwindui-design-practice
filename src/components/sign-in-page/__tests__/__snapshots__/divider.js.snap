// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider renders correctly 1`] = `
<div
  className="divider"
>
  <span />
  <p>
    Or continue with
  </p>
  <span />
</div>
`;
